.github/workflows/report_news/main.py

```diff
@@ -58,6 +62,62 @@ def fetch_tech_griot() -> list:
     return articles
 
 
+def fetch_peef():
+    """
+    Fetching data from peef
+    """
+
+    url = f'{PEEF_SITEMAP_URL}'
+
+    with urlopen(url) as response:
+        body = response.read()
+
+    xml_raw = ET.fromstring(body)
+
+    articles = []
+    for url in xml_raw.findall('{http://www.sitemaps.org/schemas/sitemap/0.9}url')[-MAX_ARTICLES:]:
+        loc = url.find('{http://www.sitemaps.org/schemas/sitemap/0.9}loc')
+        lastmod = url.find('{http://www.sitemaps.org/schemas/sitemap/0.9}lastmod')
+
+        if loc is None or lastmod is None:
+            continue
+
+        link = re.sub(r'^http://', 'https://', urlquote(loc.text, safe=":/"))
+        pub_date = lastmod.text
+
+        if not link.startswith("https://peef.dev/post/"):
```

Comment:
I would like to add here that not all content from /post/ are article

1. We have comments or simple post for the early version : `/post/<slug>` where slug is autogenerated
2. And articles are from `/post/<username>/<slug>` where slug comes from the article's title

So, i think normal` /post/<slug>` should be excluded and use only `/post/<username>/<slug>`

Reply:
Okay, I will updated 👌